src/core/server/http/integration_tests/lifecycle.test.ts

```diff
@@ -684,6 +701,27 @@ describe('Auth', () => {
     expect(response.header['www-authenticate']).toBe(authResponseHeader['www-authenticate']);
   });
 
+  it('attach security header to not handled auth response', async () => {
```

Comment:
duplicates tests in src/core/server/http/integration_tests/router.test.ts should I remove them?

Reply:
One seems enough. Not sure in which file it makes more sense to keep it though. There seems to be overlapping 'coverage' between the two test files.